docs/consensus_decisions.md

```diff
@@ -67,3 +69,19 @@ The group believes that the known value of this feature can be sufficiently cove
 The cost analysis of the nested selectors feature was performed in the absence of sufficient in-field experience of use in production systems.
 In result, the group's decision to not currently incorporate the feature is based on the lack of sufficient known value that would require them, which the group recognizes may change in the future.
 In result, it is the intent of the group to design MessageFormat 2 in a way that wouldn't prevent future revisions of the standard to be extended with nested selectors feature.
+
+## 7: A valid MessageFormat implementation may require all formatting functions to be run without access to the runtime context.
+
+**Discussion:**
+It is theoretically possible for formatting functions to become an attack vector to a system,
+as they are often handling user-controlled inputs.
+It is important to ensure that it is possible for an implementation to consider formatting functions as untrusted code,
+while not requiring that this is done.
+An implementation that does limit formatting function access is still expected to
+allow for e.g. message references and other parts of the specification to work.
+
+The "runtime context" here refers to information about the current message,
+other available messages,
+the available formatting functions,
+and any arguments or parameters that may have been made available when calling the message formatter.
+The formatting function would in all cases still have access to the explicit values and options with which it was called.
```

Comment:
I miss some bullet points summarising the first paragraph and what we are agreeing with would be very handy and clarifying, wrote my understanding about the topic, hopefully it can help 

- This consensus doesn't discard that other that future proposed solutions can be used but 

 -  We agree that all security concerns that can arise must be addressed and avoided somehow, always when possible and translatable as design constraints or recommendations from our side.
    - security levels
    - implementation constrains
    - best practices
    - others

- We must ensure that non-malicious code can be used by using one of the proposed alternatives for formatting functions:
   - no access to anything
   - limit access via context or any other interface interface
   -  ~~full access~~